drs_downloader_to_getm.wdl

```diff
@@ -159,11 +160,19 @@ task download {
             start_time=`date +%s`
             signed_urls=($(cat ~{manifest} | jq -r '.[] .url'))
             for signed_url in ${signed_urls[@]}; do
-                # this is going to create some crazy truncated names but it shouldn't make a difference in run times
-                wget ${signed_url} -P ${TMP_DL_DIR}/
+                wget ${signed_url} --output-document=${TMP_DL_DIR}/$(basename $signed_url)
             done
             end_time=`date +%s`
             total_time="$(($end_time-$start_time))"
+
+            # time just the md5sums
+            for signed_url in ${signed_urls[@]}; do
+                md5sum ${TMP_DL_DIR}/$(basename $signed_url)
```

Comment:
Conceptually `$(basename $signed_url)` makes sense, assuming that it would return the last element of the bucket key. In practice it doesn't work because the signed URL contains many subsequent elements, which confuse basename. Two options come to mind, either:
- Run md5sum on each file in `${TMP_DL_DIR}` (e.g., `${TMP_DL_DIR}/*`
- Use the `filepath` field in the getm manifest which I _think_ will match the name of the file in the ${TMP_DL_DIR}